taskcluster/ci/version-bump/kind.yml

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: fenix_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - fenix_taskgraph.transforms.multi_dep:transforms
+    - fenix_taskgraph.transforms.version_bump:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - push-apk
+
+primary-dependency: push-apk
+
+group-by: build-type
+
+only-for-build-types:
+    - beta
+    - release
+
+job-template:
+    description: Fenix version bump/tag
+    worker-type: tree
+    worker:
+        bump: true
+        bump-files: ["version.txt"]
+        push:
+            by-level:
+                "3": true
+                # If you set the following line to true, you need to grant write access
+                # to https://github.com/mozilla-release-automation-bot-staging on your
+                # fork.
```

Comment:
Doing so enabled me to get this kind of staging commits https://github.com/JohanLorenzo/fenix/commit/f493a3adc9cacd1742683e0e7a869d3f207f48c8